tools/check_test.js

```diff
@@ -30,6 +30,12 @@ function Driver() {
     fail: 0,
     skip: 0,
     timeout: 0,
+    json: {
+      bin: { text:0, total:0, data:0, bss:0, rodata:0 },
```

Comment:
@daeyeon The `bin` attribute contains information about the iotjs (or jerry) binary size, where `total` means the full binary size, the others just the main segments.
The `submodules` contains information about the git repositories (os [nuttx, tizenrt], app [jerry, iotjs]) that is used for testing. This is useful to identify the testing environment:

```json
  "submodules": {
    "nuttx": {
      "hash": "e803e2c",
      "author": "Gregory Nutt <gnutt@nuttx.org>",
      "date": "Tue Feb 7 17:16:56 2017 -0600",
      "log": "Costmetic changes from review of last PR."
    },
    "nuttx_apps": {
      "hash": "bf9ca88",
      "author": "Gregory Nutt <gnutt@nuttx.org>",
      "date": "Tue Feb 7 07:56:54 2017 -0600",
      "log": "NSH: Both arguments of 'ln' command may be relative paths"
    },
    "iotjs": {
      "hash": "8a4ccc0",
      "author": "Roland Takacs <h868310@gmail.com>",
      "date": "Thu Jan 26 08:51:21 2017 +0100",
      "log": "Correct wrong urls in README.md (#606)"
    }
  },
```